src/main/java/org/quiltmc/quiltmappings/loom/QuiltMappingsOnLoomPlugin.java

```diff
@@ -42,7 +47,7 @@
 import net.fabricmc.mappingio.tree.MemoryMappingTree;
 
 @SuppressWarnings("UnstableApiUsage")
-public class QuiltMappingsOnLoomPlugin implements Plugin<Project> {
+public class QuiltMappingsOnLoomPlugin implements Plugin<PluginAware> {
     private static final int HASH_CODE_VERSION_BASE = 1;
```

Comment:
Could you increase this by one so that users don't have to delete their cache to use the unpick layer on the same mappings version?